renderer/plugin.js

```diff
@@ -12,15 +12,29 @@ org.ekstep.questionunitmtf.RendererPlugin = org.ekstep.contentrenderer.questionU
   _render: true,
   _selectedAnswers: [],
   _dragulaContainers: [],
+  _constant: {
+    horizontal: "Horizontal",
+    vertial : "Vertical"
+  },
   setQuestionTemplate: function () {
-    this._question.template = QS_MTFTemplate.htmlLayout;
+    MTFController.initTemplate(this);// eslint-disable-line no-undef
   },
   preQuestionShow: function (event) {
     this._super(event);
+    var templateTypeContent;
+    if (this._question.config.layout == this._constant.vertial) { // eslint-disable-line no-undef
+      templateTypeContent = MTFController.getVerticalLayout;      
```

Comment:
MTFController.getVerticalLayout is a funciton not a variable.

Reply:
Yes Vinu. `MTFController.getVerticalLayout` is a function. This function is being called below based on the selected layout.